inc/field/fieldsfield.class.php

```diff
@@ -152,16 +149,24 @@ public function getBlocks(): array {
    }
 
    public function showForm(array $options): void {
+      if (!\Plugin::isPluginActive('fields')) {
+         $options['error'] = __('Warning: Additional Fields plugin is disabled or missing', 'formcreator');
+         $template = '@formcreator/field/undefinedfield.html.twig';
+         TemplateRenderer::getInstance()->display($template, [
+            'item' => $this->question,
+            'params' => $options,
+         ]);
+         return;
+      }
```

Comment:
show an error message if ADditioinal fields is disabled, while allowing to edit the question